### PR TITLE
Trade client

### PR DESCRIPTION
**Pull request by NylonEric**
* add color change and validation function
* validation and message working
* client trade messaging working

### CODE REVIEW

File: client/src/components/Trade/Trade.jsx [hunk 1]
```diff
@@ -26,35 +26,80 @@ class Trade extends React.Component {
...
+    const stockOwned = currentUser !== undefined ? currentUser.userPortfolio.filter(stock => {
+      return stock.ticker_symbol === stockSymbol;
+    }) : 0;
+    const price = this.props.stockSelected.price;
+    const cashBalance = (this.props.user.cashBalance/100).toFixed(2);
+    const action = this.props.action;
+
+    if (tradeValidation(shares, stockOwned, price, cashBalance, action)) {
+      this.setState({
+        [name]: value,
```
Comment (tbrinkman3) [inline client/src/components/Trade/Trade.jsx:44]: This allows you to declare a variable as a key name, correct?

File: client/src/components/Trade/helperFunctions/tradeValidation.js [hunk 2]
```diff
@@ -1,3 +1,13 @@
-module.exports.tradeValidation = (shares, price, cashBalance, action) => {
-  return true;
+module.exports.tradeValidation = (shares, stockOwned,  price, cashBalance, action) => {
+  console.log('in tradeValidation function', shares, price, cashBalance, action);
```
Comment (tbrinkman3) [inline client/src/components/Trade/helperFunctions/tradeValidation.js:2]: Still needed?

**Review by tbrinkman3 — APPROVED** [review]
Looks good!

File: client/src/components/Trade/Trade.jsx (hunk 1 above)
Comment (NylonEric) [reply client/src/components/Trade/Trade.jsx:44]: Yep!

**Pull request merged**